payjoin/src/receive/v2/mod.rs

```diff
@@ -240,19 +240,19 @@ impl Receiver<UninitializedReceiver> {
         };
         MaybeBadInitInputsTransition::success(
             SessionEvent::Created(session_context.clone()),
-            Receiver { state: WithContext { context: session_context } },
+            Receiver { state: Initialized { context: session_context } },
         )
     }
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-pub struct WithContext {
+pub struct Initialized {
```

Comment:
Nit the commit says
>shared SessionContext can be moved to Receiver and out of state field.

Is this justiification for the rename or something that should be done in this commit?

Reply:
Another PR that is coming imminently since it does affect the Serialization